src/components/common/GlobalNav/GlobalNav.js

```diff
@@ -1,19 +1,18 @@
 import { Box, Grid, Typography } from '@material-ui/core';
-import Link from 'next/link';
-import { useRouter } from 'next/router';
+import { Link } from 'gatsby';
 import React from 'react';
 import { createUseStyles } from 'react-jss';
 import { PAGELIST } from '../../../../data/templateMeta';
 
-const GlobalNav = () => {
+const GlobalNav = props => {
   const classes = styles(),
-    router = useRouter(),
+    { location } = props,
```

Comment:
This is a required prop for GlobalNav. We should probably move this component & the DefaultLayout to a `layout/` dir.

Reply:
Also add prototypes & defaultprops